### PR TITLE
Styling (Closes #3)

### LINKED ISSUE #3: Add a footer with contact info
Suggested steps:

- [x] 1. Make a new branch called `footer`
- [x] 2. Create a template file at `blog/templates/footer.html` and add some markup for the footer. [Use the HTML5 `footer` tag](https://www.w3schools.com/tags/tag_footer.asp) so that accessibility software knows it's a footer. We'll probably also want to [use a flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) to arrange the icons.
- [x] 3. [Add font-awesome icons](https://docs.fontawesome.com/web/add-icons/how-to) to social media logos, a mail icon, etc. in our contact info
- [x] 4. [Add a WCAG 2.1 AA logo](https://www.w3.org/WAI/standards-guidelines/wcag/conformance-logos/#wcag-21-level-aa-conformance) to show off that we have an accessible website
- [x] 5. Update the base template `blog/templates/base.html` to include the footer. [The most straightforward way to do this is probably with `{% include 'footer.html' %}`](https://ttl255.com/jinja2-tutorial-part-6-include-and-import/), but if you find something else that works better, go for it
- [x] 6. Optional: update `blog/static/style.css` with any footer-specific styling you'd like
- [x] 7. Run the app and make sure it looks right. Open the dev console `F12` and check for any warnings
- [x] 8. Extra credit: audit the page for accessibility using [WAVE](https://addons.mozilla.org/en-US/firefox/addon/wave-accessibility-tool/) and fix any issues you might have introduced.
- [x] 9. Create a pull request and ask someone for a code review

Acceptance criteria:

- [x] The footer displays our Twitter and Instagram links and our email address as a mailto link
- [x] Footer markup is in its own template file
- [ ]  `tests/test_blog.py` is updated to check for contact info links (see the `test_index` function for an example)

### PR DESCRIPTION
**Pull request by junkshark8**
Styled nav and made it responsive, styled pages and added some buttons & icons for usability, created footer with social/email links

### CODE REVIEW

File: .DS_Store
Comment (ada-fruit) [inline .DS_Store:1]: yucky, i'll add these to `.gitignore`

**Review by ada-fruit — CHANGES_REQUESTED** [review]

**Comment by ada-fruit** [comment]
also: Closes #3

**Comment by ada-fruit** [comment]
actions plz

File: blog/static/favicon.ico
Comment (ada-fruit) [inline blog/static/favicon.ico:1]: also, what is this diff? i'm assuming you just fixed the dimensions? (i made the old favicon in like 5 minutes, so i would not be offended if you wanted to replace it)

Comment (junkshark8) [reply blog/static/favicon.ico:1]: It had a transparent bg and looked weird in my browser

File: blog/static/colors.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+:root {
```
[the remark this answers is not in the document]
Comment (junkshark8) [reply blog/static/colors.css:1]: gonna do [this](https://stackoverflow.com/questions/50910404/best-practice-for-naming-css-color-variables#:~:text=Or%20by%20their%20meaning%2Ffunction,%2C%20disabled%2Dcolor%20...&text=meaning%2Ffunction%20is%20the%20best,and%20changes%20on%20a%20site.)

[the remark this answers is not in the document]
Comment (ada-fruit) [reply blog/static/colors.css:1]: even better, thanks for the research

File: blog/templates/base.html [hunk 2]
```diff
@@ -4,20 +4,47 @@
...
-        <ul>
-            {% if g.user %}
-            <li><span>{{ g.user['username'] }}</span></li>
-            <li><a href="{{ url_for('auth.logout') }}">Log Out&nbsp;<i class="fa-solid fa-right-from-bracket"></i></a></li>
-            {% else %}
-            <li><a href="{{ url_for('auth.login') }}">Log In&nbsp;<i class="fa-solid fa-right-to-bracket"></i></a></li>
-            {% endif %}
-        </ul>
+        <!-- logo next to header, links to home -->
+        <a class="logo" href="" aria-label="Home">
```
Comment (ada-fruit) [inline blog/templates/base.html:24]: should be `href="/"` to point to the main page

**Review by ada-fruit — CHANGES_REQUESTED** [review]

**Review by ada-fruit — APPROVED** [review]

**Pull request merged**